homeassistant/components/dsmr/const.py

```diff
@@ -245,10 +245,28 @@
         state_class=SensorStateClass.MEASUREMENT,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    DSMRSensorEntityDescription(
+        key=obis_references.BELGIUM_MAX_POWER_PER_PHASE,
+        name="Max power per phase",
+        dsmr_versions={"5B"},
+        device_class=SensorDeviceClass.POWER,
+        entity_registry_enabled_default=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    DSMRSensorEntityDescription(
+        key=obis_references.BELGIUM_MAX_CURRENT_PER_PHASE,
+        name="Max current per phase",
+        dsmr_versions={"5B"},
+        device_class=SensorDeviceClass.POWER,
+        entity_registry_enabled_default=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
     DSMRSensorEntityDescription(
         key=obis_references.ELECTRICITY_IMPORTED_TOTAL,
         name="Energy Consumption (total)",
-        dsmr_versions={"5", "5B", "5L", "5S", "Q3D"},
+        dsmr_versions={"5L", "5S", "Q3D"},
```

Comment:
This also removed "5", which is used in NL. I don't think that is correct?

Reply:
According to this website, this parameter is also not present in the dutch variant:
https://www.netbeheernederland.nl/_upload/Files/Slimme_meter_15_32ffe3cc38.pdf

To clarify, BE and NL are identical but BE supports current limiting and a switch for E and valve for G

Reply:
Extra info: ELECTRICITY_IMPORTED_TOTAL is represented by OBIS reference 1-0:1.8.1 (cfr https://github.com/ndokter/dsmr_parser/blob/master/dsmr_parser/obis_references.py#L64)

The pdf from netbeheernederland.nl does not mention OBIS reference 1-0:1.8.1 in the example telegrams.

Reply:
👍 Thanks for clarifying, checked it indeed with 3 different meters in NL, they are indeed missing and not in the specifications.

Thanks 👍